charts/argo-cd/values.yaml

```diff
@@ -1891,7 +1892,7 @@ applicationSet:
     # -- Repository to use for the application set controller
     repository: quay.io/argoproj/argocd-applicationset
     # -- Image pull policy for the application set controller
-    pullPolicy: IfNotPresent
+    imagePullPolicy: IfNotPresent
```

Comment:
What about also usint the global by default and setting this to empty string?

Reply:
The reason why I haven't done that is because of repository that differs from what's in the global section. If you want I can change this. Is there any plan to switch repository / command to applicationset binary in the upstream image?

Reply:
Look at this [diff](https://github.com/argoproj/argo-helm/compare/master...pdrastil:feature/image?expand=1) where you can see what it would mean to change repository and let me know if you want this as a separate PR or in this one.